frontend/packages/ceph-storage-plugin/src/components/modals/volume-snapshot-modal/volume-snapshot-modal.tsx

```diff
@@ -208,8 +208,8 @@ export const VolumeSnapshotModal = withHandlePromise((props: VolumeSnapshotModal
       const patch = [
         {
           op: 'add',
-          path: '/metadata/labels/thislabel',
-          value: `schedule-${pvcLabel}`,
+          path: '/metadata/labels',
+          value: { ...resource.metadata.labels, [pvcLabel]: `schedule-${pvcLabel}` },
```

Comment:
op: resource.metadata.labels ? 'replace' : 'add',

Reply:
@gnehapk have changed `add` to `replace`,  we have to replace the complete labels objects because the key of the label is also variable, therefore, `add` option is not required.